presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -237,47 +253,527 @@ public void testWithLimitClause()
 
     // TODO: Handle table alias rewrite for view definition and base query https://github.com/prestodb/presto/issues/16404#issue-940248564
     @Test
-    public void testWithTableAliasInView()
+    public void testWithTableAlias()
     {
         String originalViewSql = format("SELECT base1.a, b, c FROM %s base1", BASE_TABLE_1);
         String baseQuerySql = format("SELECT a, c FROM %s", BASE_TABLE_1);
 
         assertOptimizedQuery(originalViewSql, baseQuerySql, baseQuerySql);
-    }
 
-    @Test
-    public void testWithTableAliasInBaseQuery()
-    {
-        String originalViewSql = format("SELECT a, b, c FROM %s", BASE_TABLE_1);
-        String baseQuerySql = format("SELECT base1.a, c FROM %s base1", BASE_TABLE_1);
+        originalViewSql = format("SELECT a, b, c FROM %s", BASE_TABLE_1);
+        baseQuerySql = format("SELECT base1.a, c FROM %s base1", BASE_TABLE_1);
 
         assertOptimizedQuery(originalViewSql, baseQuerySql, baseQuerySql);
     }
 
     @Test
     public void testWithJoinTables()
     {
-        String originalViewSql = format("SELECT %s.a, %s.b FROM %s JOIN %s ON %s.c = %s.c", BASE_TABLE_1, BASE_TABLE_2, BASE_TABLE_1, BASE_TABLE_2, BASE_TABLE_1, BASE_TABLE_2);
+        String originalViewSql = format("SELECT %s.a, %s.b FROM %s JOIN %s ON %s.c = %s.c",
```

Comment:
keep format in separate line for cases when overall function is breaking into multiple lines